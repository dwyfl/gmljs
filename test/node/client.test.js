import GMLClient from '../../src/node/client';

it('creates a correct default GMLClient node', () => {
  const gml = GMLClient.create().toString();
  expect(gml).toMatchSnapshot();
});