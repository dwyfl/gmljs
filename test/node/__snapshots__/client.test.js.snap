// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates a correct default GMLClient node 1`] = `"<client><name>GML.js</name><version>1.0</version><time>1507479541</time></client>"`;
